sechub-doc/src/docs/asciidoc/documents/other/review-guide.adoc

```diff
@@ -0,0 +1,487 @@
+:toc: left
+== Reviewers guide
+
+At {sechub} we have some quality aspects for pull requests.
```

Comment:
Once cloud also add a PR Template on the Repo level which includes some checkboxes to ensure the Author thought about some crucial points (Based on this Doc).

I have done it my current projects at work and it really helped me reduce some noise around Code Reviews and PRs.

See: [PR Template Guide](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

Reply:
good idea, thank you